pages/apiPages/message_page.ts

```diff
@@ -86,7 +88,7 @@ export class ApiMessage3003Page {
             data,
             headers: headers,
         })
-        expect(apiRequest.ok()).toBeTruthy()
+        // expect(apiRequest.ok()).toBeTruthy()
```

Comment:
@PikachuSerg1 please remove comment // and left the assertion

Reply:
Updated